tests/test_valkey_subscriber.py

```
@@ -1,5 +1,7 @@
 from datetime import UTC, datetime, timedelta
-from unittest.mock import MagicMock
+from unittest.mock import AsyncMock, MagicMock, patch
+
+import pytest
```

Flake8 (7.3.0)

[error] 'pytest' imported but unused

(F401)